backend/globaleaks/models/__init__.py

```diff
@@ -874,7 +936,7 @@ class _Subscriber(Model):
     surname = Column(UnicodeText, nullable=False)
     phone = Column(UnicodeText, default='', nullable=False)
     email = Column(UnicodeText, nullable=False)
-    organization_name = Column(UnicodeText, default='', nullable=False)
+    organization_name = Column(UnicodeText, default='', nullable=False, unique=True)
```

Comment:
This change might not be included as known to have caused some problems:
- If an organization register and do not get approved they wont be able to register again.
- If a fake user user attempt a registration on behalf of a real organization, the real organization wont be able to register again